packages/features/src/send/components/ConfirmTransactionForm.tsx

```diff
@@ -68,7 +76,18 @@ export const ConfirmTransactionForm = () => {
         : Mina.TransactionKind.PAYMENT
     )
     const getPassphrase = async () => Buffer.from(data.spendingPassword)
-    const signedTx = await sign(constructedTx as any, getPassphrase)
+    let signedTx
+    try {
+      signedTx = await sign(constructedTx as any, getPassphrase)
```

Comment:
As we've mentioned, looking ahead, we can spend some time working on improving the situation with these `any` types by building out a good multi-chain abstraction layer  😄